bokeh/models/plots.py

```diff
@@ -737,6 +737,15 @@ class _list_attr_splat(list):
     def __setattr__(self, attr, value):
         for x in self:
             setattr(x, attr, value)
+    def __getattribute__(self, attr):
+        if len(self) == 0:
+            raise AttributeError("Trying to access %r attribute on an empty 'splattable' list" % attr)
+        if len(self) == 1:
+            return getattr(self[0], attr)
+        try:
+            return _list_attr_splat([getattr(x, attr) for x in self])
+        except:
```

Comment:
`except Exception` is the preferred form. Otherwise system exceptions are going to be caught here as well.

Reply:
Oh, thank you (coding in a hurry)

Reply:
If there is a specific exception to consider (`AttributeError`?), that would be even better.